index.html

```diff
@@ -53,6 +54,241 @@ <h1>I'm Redwan.<br/>
+                        </div>
+                    </div>
+                </div>
+               </div>
+        </section>
+
+
+        <section class="section-bottom"> 
+            <div class="headerTop">
+                <h1>About Myself </h1>
+                <p >
+                    Hello I’m a software developer! I can 
+                    help you build a product , feature or 
+                    website. Look through some of my work 
+                    and experience! If you like what you see 
+                    and have a project you need coded, 
+                    don’t hesitate to contact me.
+                </p>
+            </div>
+            <!-- End of header top -->
+            <div class="headerBottom">
+                <a href="#">LET’S CONNECT</a>
+                <div class="link-btn">
+                    <ul>
+                        <li>
+                            <a href="#"><img src="images/link-Vector4.svg" alt=""/></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/Instagram2.png" alt=""/></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/link-Vector5.svg" alt=""/></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/Twitter.svg" alt="Twitter"/></a>
+                        </li>
+                        
+                        <li>
+                            <a href="#"><img src="images/link-Vector3.svg" alt=""/></a>
+                        </li>
+                        
+                       
+                    </ul>
+                </div>
+                <div class="action">
+                    <button class="submit">Get my resume</button>
+                </div>
+                <div class="tags-bottom">
+                    <div class="container-links">
+                        <ul>
+                            <li> Languages</li>
+                        </ul>
+                    </div>
+                    
+                    <div class="links">
+                        <div class="btn">
+                            <div class="link-log"><img src="Images/JS.png" alt="">  </div>
+                            <div class="lnk-txt"><a  href="">JavaScript</a></div> 
+                        </div>
+                        <div class="btn">
+                            <div class="link-log"><img src="Images/HTML.png" alt="">  </div>
+                            <div class="lnk-txt"><a  href="">HTML</a></div> 
+                        </div>
+                        <div class="btn">
+                            <div class="link-log"><img src="Images/CSS.png" alt="">  </div>
+                            <div class="lnk-txt"><a  href="">CSS</a></div> 
+                        </div>
+                    </div>
```

Comment:
- [ ] Kindly wrap this section with **ul** instead. Because it is a list 🔢 Let's use all the tags properly 🤗